Cargo.toml

```diff
@@ -1,23 +1,23 @@
 [workspace]
 
 members = [
-    "crates/acvm_backend_barretenberg",
-    "crates/noirc_evaluator",
-    "crates/noirc_frontend",
-    "crates/noirc_errors",
-    "crates/noirc_driver",
-    "crates/noirc_printable_type",
-    "crates/nargo",
-    "crates/nargo_cli",
-    "crates/nargo_toml",
-    "crates/fm",
-    "crates/arena",
-    "crates/noirc_abi",
-    "crates/noirc_abi_wasm",
-    "crates/iter-extended",
-    "crates/wasm",
+    "compiler/noirc_evaluator",
+    "compiler/noirc_frontend",
+    "compiler/noirc_errors",
+    "compiler/noirc_driver",
+    "compiler/noirc_printable_type",
+    "tooling/acvm_backend_barretenberg",
+    "tooling/nargo",
+    "tooling/nargo_cli",
+    "tooling/nargo_toml",
+    "compiler/fm",
+    "compiler/arena",
+    "compiler/noirc_abi",
+    "compiler/noirc_abi_wasm",
+    "compiler/iter-extended",
```

Comment:
Can we have a `utils/` for things like these?

Reply:
I'm on the fence about a folder named utils,  I've been against a utils `crate` since it can become a place where you put anything that you don't really know where it belongs, a utils folder with clearly defined crates in it doesn't sound bad

Reply:
Since moving these doesn't require us to move around a lot of code, lets try a utils folder and revert if it starts to look unwieldly